ctags/symbol_kind.go

```diff
@@ -0,0 +1,110 @@
+package ctags
+
+import "strings"
+
+type SymbolKind uint8
+
+const (
+	Accessor SymbolKind = iota
```

Comment:
Working on this PR got me thinking ... it'd be nice to just use SCIP as the exchange format instead of universal-ctags output, and have a tool to map universal-ctags onto SCIP. That would get us closer to an actual spec, unlike the ctags output which has inconsistent naming and is not guaranteed to be stable.

Reply:
100% agree with this. I think this was discussed when scip-ctags was added and that was seen as an part of the end goal.